libs/remix-ui/git/src/components/panels/tokenWarning.tsx

```diff
@@ -1,12 +1,20 @@
 import { gitPluginContext } from "../gitui"
-import React, { useEffect, useState } from "react";
-export const TokenWarning = () => {
+import React, { useEffect, useState } from "react"
+
+export interface TokenWarningProps {
+  plugin: any
```

Comment:
plugin is in the context no need to pass it around

Reply:
100%. You are right.